From de06aad620f7c6751a760c5b8fcc13fd1e163f73 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:08:23 -0400
Subject: [usb] sierra: remove unused variable

Message-id: <1410887463-20674-432-git-send-email-dzickus@redhat.com>
Patchwork-id: 95344
O-Subject: [RHEL7 PATCH 431/591] USB: sierra: remove unused variable
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit bc03cfe84d833a49a0a0a6d5e45fc84169f48e30
Author: Johan Hovold <jhovold@gmail.com>
Date:   Mon May 26 19:23:00 2014 +0200

    USB: sierra: remove unused variable

    Remove unused variable from sierra_release_urb.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/sierra.c b/drivers/usb/serial/sierra.c
index 169899f..fa0b78a 100644
--- a/drivers/usb/serial/sierra.c
+++ b/drivers/usb/serial/sierra.c
@@ -419,9 +419,7 @@ static int sierra_tiocmset(struct tty_struct *tty,
 
 static void sierra_release_urb(struct urb *urb)
 {
- struct usb_serial_port *port;
  if (urb) {
-  port = urb->context;
   kfree(urb->transfer_buffer);
   usb_free_urb(urb);
  }
-- 
1.7.1